website/docs/r/appautoscaling_scheduled_action.html.markdown

```diff
@@ -87,3 +87,11 @@ The following arguments are supported:
 In addition to all arguments above, the following attributes are exported:
 
 * `arn` - The Amazon Resource Name (ARN) of the scheduled action.
+
+## Import
```

Comment:
This documentation was added to the `aws_appautoscaling_scheduled_action` resource page instead of `website/docs/r/autoscaling_schedule.html.markdown` 😄

Reply:
Oh, I am sorry for the poor mistake..

Reply:
Fixed in 850642a